src/tests/base_kubeapi_test.py

```diff
@@ -125,6 +125,7 @@ def _wait_for_install(cls, agent_cluster_install: AgentClusterInstall, agents: L
         agent_cluster_install.wait_to_be_installing()
         Agent.wait_for_agents_to_install(agents)
         agent_cluster_install.wait_to_be_installed()
+        agent_cluster_install.download_kubeconfig()
```

Comment:
isn't ``kubeconfig_path`` a required argument?

Reply:
oh right, assumed it's the same as cluster.download_kubeconfig: https://github.com/openshift/assisted-test-infra/blob/6aaf9a3f08d7e9f17ff281c8cfdf29f55a8a5b9d/src/assisted_test_infra/test_infra/helper_classes/cluster.py#L908